libs/blocks/merch/checkout-context.js

```diff
@@ -0,0 +1,30 @@
+const DEFAULT_WORKFLOW_STEP = 'email';
```

Comment:
since both these variables are only being used in one place you don't need to make them global. It's extra bytes for no reason .

Reply:
sure, will fix that